content/participants/ttjc@KMuthusamyms/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Muthusamy
+type: "participant"
+name: "Muthusamy K"
+portfolio: "https://kmuthusamyms.github.io"
+github-link: "https://github.com/KMuthusamyms/"
+linkedin-link: "https://linkedin.com/in/muthusamyms"
+looking-for: "job"
+description: "Machine Learning and Fullstack development enthusiast."
+---
+
+Hi! I am Muthusamy, let me walk you through my intro, projects and blogs.  
+
+### introduction
+
+I am a Software Systems Student from Kamaraj College of Engineering and Technology, Madurai. The expected pass out is July 2020. I like to learn the new cutting edge technologies using the learn by doing approach. I like to work with fullstack developments and javascript frameworks.   
+
+## why are you taking this challenge?
+
+I looking forward to kickstart my career as a software developer or fullstack developer. 
+And also I want to learn new technologies for my skill growth.
+
+## something interesting about you?
+
+I will spend every sunday to do some social works like tree plantations, cleaning public places.  
+
+
+### help I need?
+
+I know the technologies like NodeJS, AngularJS, MySQL, MongoDB, etc. But I want to learn how these technologies are used in big organisations and know what should I have to improve myself to become good software developer.    
+
+### how to contact me?
+
+- [LinkedIn](https://www.linkedin.com/in/muthusamyms)
+
+### projects
```

Comment:
It's good that you have listed your projects but can you please provide the source code and hosted link of the projects. Also projects after mid-March are accepted for the challenge so update accordingly.

Reply:
> It's good that you have listed your projects but can you please provide the source code and hosted link of the projects. Also projects after mid-March are accepted for the challenge so update accordingly.

I have made changes.